automated/linux/hci-smoke/hci-smoke.yaml

```diff
@@ -13,14 +20,19 @@ metadata:
         - functional
     devices:
         - dragonboard410c
+        - imx8mmevk
+        - imx6ullevk
+        - rpi4
 
 params:
     DEVICE: hci0
     # expected state on first boot: <enabled> or <disabled> or <auto>
     BOOT: enabled
+    # MODE can be set to "normal", "le" or "both"
+    MODE: normal
```

Comment:
I think it would be just simpler if MODE includes 'scan' or 'lescan' instead of normal, le, both. And MODE should be SCAN instead.

Reply:
Do you ask to remove 'both' or just rename 'normal' to 'scan'?

Reply:
yes, remove 'both', and have 'scan' and 'lescan' as the 2 supported modes. Then the tester can use MODE='scan' or MODE='scan lescan'

Reply:
OK